packages/calendar/test/calendar.test.ts

```diff
@@ -10,23 +10,67 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
-import sinon, { spy } from 'sinon';
-
+import { sendKeys, sendMouse } from '@web/test-runner-commands';
 import { testForLitDevWarnings } from '../../../test/testing-helpers.js';
-import { Calendar } from '@spectrum-web-components/calendar';
-
+import { spy } from 'sinon';
+import {
+    CalendarDate,
+    endOfMonth,
+    isSameDay,
+    parseDate,
+    today,
+} from '@internationalized/date';
+import { Button } from '@spectrum-web-components/button';
+import { Calendar, DAYS_PER_WEEK } from '@spectrum-web-components/calendar';
 import '@spectrum-web-components/calendar/sp-calendar.js';
 import '@spectrum-web-components/theme/sp-theme.js';
+import { spreadProps } from '../../../test/lit-helpers.js';
 
-const DEFAULT_LOCALE = 'en-US';
-const CALENDAR_TITLE_SELECTOR = '[data-test-id="calendar-title"]';
+const LOCAL_TIME_ZONE = new Intl.DateTimeFormat().resolvedOptions().timeZone;
 const NEXT_BUTTON_SELECTOR = '[data-test-id="next-btn"]';
 const PREV_BUTTON_SELECTOR = '[data-test-id="prev-btn"]';
-const FIRST_ACTIVE_DAY_SELECTOR =
-    '[data-test-id="calendar-day"]:not(:is(.is-disabled, .is-outsideMonth))';
+
+async function fixtureElement({
+    locale = 'en-US',
+    props = {},
+}: {
+    locale?: string;
+    props?: { [prop: string]: unknown };
+} = {}): Promise<Calendar> {
+    const wrapped = await fixture<HTMLElement>(html`
+        <sp-theme lang=${locale} color="light" scale="medium">
+            <sp-calendar ...=${spreadProps(props)}></sp-calendar>
+        </sp-theme>
+    `);
+    const el = wrapped.querySelector('sp-calendar') as Calendar;
+    await elementUpdated(el);
+    return el;
+}
 
 describe('Calendar', () => {
-    const sandbox = sinon.createSandbox();
+    let element: Calendar;
+    const originalDateNow = Date.now;
+    const fixedYear = 2022;
+    const fixedMonth = 5;
+    const fixedDay = 15;
+
+    before(async () => {
+        const fixedTime = new Date(
+            fixedYear,
+            fixedMonth - 1, // 0-indexed in Date but 1-indexed in CalendarDate
+            fixedDay
+        ).getTime();
+        Date.now = () => fixedTime;
+    });
+
+    beforeEach(async () => {
+        element = await fixtureElement();
+        await elementUpdated(element);
+    });
+
+    after(() => {
+        Date.now = originalDateNow;
```

Comment:
If sinon is used in these tests, should we also use a sandbox to stub the `Date.now` function?